src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -346,11 +346,11 @@ void DoorLockServer::setUserCommandHandler(chip::app::CommandHandler * commandOb
                                            const chip::app::Clusters::DoorLock::Commands::SetUser::DecodableType & commandData)
 {
     auto & userIndex = commandData.userIndex;
-    emberAfDoorLockClusterPrintln("[SetUser] Incoming command [endpointId=%d,userIndex=%d]", commandPath.mEndpointId, userIndex);
+    ChipLogProgress(Zcl, "[SetUser] Incoming command [endpointId=%d,userIndex=%d]", commandPath.mEndpointId, userIndex);
 
     if (!SupportsUSR(commandPath.mEndpointId))
     {
-        emberAfDoorLockClusterPrintln("[SetUser] User management is not supported [endpointId=%d]", commandPath.mEndpointId);
+        ChipLogProgress(Zcl, "[SetUser] User management is not supported [endpointId=%d]", commandPath.mEndpointId);
```

Comment:
Wondering whether this and some of the other error cases here should be ChipLogError.  Followup for that, though.

Reply:
Yes, a lot of errors were just passed into Println (which went nowhere it seems, but even once enabled they would do progress and that is wrong). We will have to followup if we want to move detail and error.